cli.js

```diff
@@ -7,14 +7,30 @@ const cli = meow(`
 	Usage
 	  $ latest-version <package-name>
 
+	Options
+	  -v         Ouput the version of this package
+	  --version  Specify which dist-tag to use to find the latest version
```

Comment:
The description here could be better. It's not just the dist-tag, but it could also be a semver-range.

Reply:
Good point.